src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -166,10 +166,10 @@
       [:div {}
        [comps/Button {:text "Sign In"
                       :href (str "/service/login?path="
-                                 (js/encodeURIComponent (let [hash (nav/get-hash-value)]
+                                 (js/encodeURIComponent (js/decodeURIComponent (let [hash (nav/get-hash-value)]
```

Comment:
After poking at this for a bit - yes, the issue appears to be in orchestration, specifically lines 188-195 of OAuthService.scala. But it's happening because of built-in rules of spray's Uri, and it's surprisingly hard to override. Meanwhile, I don't think this current PR is actually going to cause any problems - since this is all in the fragment, any character is valid.